msu/squirrel_hooks/mod_hooks.nut

```diff
@@ -140,7 +168,7 @@ local _mods_runQueue = ::_mods_runQueue;
 					else if (compareVersions(dependencyTable, dependencyMod))
 					{
 						local tableVersion = dependencyTable.SemVer == null ? dependencyTable.Version : ::MSU.SemVer.getVersionString(dependencyTable.SemVer);
-						::MSU.QueueErrors.add(mod.FriendlyName + " is not compatible with " + dependencyMod.FriendlyName + " version " + dependencyTable.VersionOperator + tableVersion);
+						::MSU.QueueErrors.add(mod.FriendlyName + " is not compatible with " + dependencyMod.FriendlyName + " version " + getTextForOperator(dependencyTable.VersionOperator) + " " + tableVersion);
```

Comment:
Make version plural so it's `versions`, that makes more sense if we're structuring the sentence this way now.
eg:
`Mod X is not compatible with Mod Y versions older than 1.0.0`
`Mod A is not compatible with Mod Y versions up to 2.0.0`

Reply:
But that will not work with situations where it says `=` or `!=` e.g.
`Mod x is not compatible with Mod Y versions 1.2.0` - makes no sense. So I'd have to wrap the word "version" within the `getTextForOperator` function and return singular/plural for different cases.